tests/test_deprecations.py

```diff
@@ -33,18 +42,33 @@ def test_renamed_surrogate():
 
 
 def test_missing_strategy_type(config):
-    """Specifying a strategy without a corresponding type raises a warning."""
+    """Specifying a recommender without a corresponding type raises a warning."""
```

Comment:
Shouldn't the test itself also be renamed?

Reply:
absolutely, done